vmdb/Gemfile

```diff
@@ -9,16 +9,15 @@ MiqBundler.include_gemfile("../lib/Gemfile", binding)
 
 gem "jquery-rjs", "=0.1.1", :git => 'https://github.com/amatsuda/jquery-rjs.git'
 gem 'angularjs-rails', '=1.2.4'
-gem 'jquery-rails', "~>2.2.2"
+gem 'jquery-rails', "=2.2.2"
```

Comment:
@skateman It's totally OK from the build perspective but please confirm you really need to lock jquery-rails to =2.2.2 ?